cmd/kpod/common.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	is "github.com/containers/image/storage"
+	"github.com/containers/storage"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
+)
+
+func getStore(c *cli.Context) (storage.Store, error) {
+	options := storage.DefaultStoreOptions
+	if c.GlobalIsSet("root") || c.GlobalIsSet("runroot") {
```

Comment:
@nalind I think we should just eliminate these, since we probably don't want every kpod command to modify this.